slsim/Pipelines/halos_pipeline.py

```diff
@@ -0,0 +1,171 @@
+import os
+from skypy.pipeline import Pipeline
+import slsim
+import tempfile
+from astropy.cosmology import default_cosmology
+
+
+class HalosSkyPyPipeline:
+    """Class for halos configuration."""
+
+    def __init__(
+        self,
+        skypy_config=None,
+        sky_area=None,
+        m_min=None,
+        m_max=None,
+        z_max=None,
+        cosmo=None,
+        sigma_8=0.81,
+        n_s=0.96,
+        omega_m=None,
+    ):
+        """Initialize the class with the given parameters.
+
+        :param skypy_config: Path to SkyPy configuration yaml file. If None, the default SkyPy configuration file is used.
+        :type skypy_config: str or None, optional
+        :param sky_area: Sky area over which Halos are sampled. Must be in units of solid angle.
+        :type sky_area: `~astropy.units.Quantity`, optional
+        :param m_min: Minimum halo mass.
+        :type m_min: float, optional
+        :param m_max: Maximum halo mass.
+        :type m_max: float, optional
+        :param z_max: Maximum redshift value in z_range.
+        :type z_max: float, optional
+        :type sigma_8: float, optional
+        :type n_s: float, optional
+        :type omega_m: float, optional
+        :param sigma_8: matter density fluctuations on a (8 h-1 Mpc), defaults to 0.81 if not
+            specified.
+        :param n_s: Spectral index, defaults to 0.96 if not specified.
+        :param omega_m: Omega_m in Cosmology, defaults to none which will lead to the same
+            in Cosmology setting.
+        """
+
+        path = os.path.dirname(slsim.__file__)
+        module_path, _ = os.path.split(path)
+        if skypy_config is None:
```

Comment:
You have given default sky area below but I would be good to mention the default sky area here.